tests/src/common/ibc.rs

```diff
@@ -147,7 +147,7 @@ pub(super) fn send_error(
 }
 
 fn send_blank_response(app: &mut App, addr: Addr) -> ResponseWithInterChainMsgs<'_, AppResponse> {
-    send_response(app, addr, Binary(Vec::new()))
+    send_response(app, addr, Binary::new(Vec::new()))
```

Comment:
Clippy mentions that the shorthand is preferred when ran in pedantic mode and/or with `nursery` lints.
```suggestion
    send_response(app, addr, Binary::new(vec![]))
```